app/controllers/admin/metrics_controller.rb

```diff
@@ -104,7 +104,7 @@ def question_routing_guardrails_failures
   end
 
   def topics
-    scope = AnswerAnalysis.joins(:answer)
+    scope = AnswerAnalysis::AnswerTopics.joins(:answer)
```

Comment:
```suggestion
    scope = AnswerAnalysis::Topics.joins(:answer)
```

I think we don't need the Answer prefix once we're in the AnswerAnalysis namespace